feather/server/src/packet_handlers/inventory.rs

```diff
@@ -9,6 +9,7 @@ use crate::{ClientId, Server};
 pub fn handle_creative_inventory_action(
     player: EntityRef,
     packet: CreativeInventoryAction,
+    server: &mut Server,
 ) -> SysResult {
     if *player.get::<Gamemode>()? != Gamemode::Creative {
         bail!("cannot use Creative Inventory Action outside of creative mode");
```

Comment:
Aren't creative inventory action used to pick items in survival (middle click select hotbar).

Reply:
Isn't that [Pick Item](https://wiki.vg/Protocol#Pick_Item)?

Reply:
Yes, though it was a creative action for a moment, ready to merge then?